perses/rjmc/topology_proposal.py

```diff
@@ -1216,18 +1200,10 @@ def _construct_atom_map(self,
         old_res_name = old_res.name
         new_res_name = new_res.name
 
-        #make correction for HIS
-        his_templates = ['HIE', 'HID']
-        if old_res_name in his_templates:
-            old_res_name = 'HIS'
-        if new_res_name in his_templates:
-            new_res_name = 'HIS'
-        else:
-            pass
-
         current_residue_pdb_filename = resource_filename('perses', os.path.join('data', 'amino_acid_templates', f"{old_res_name}.pdb"))
         proposed_residue_pdb_filename = resource_filename('perses', os.path.join('data', 'amino_acid_templates', f"{new_res_name}.pdb"))
-
+        print(f"is the old res {old_res_name} pdb present? ", os.path.isfile(current_residue_pdb_filename))        
+        print(f"is the new res {new_res_name} pdb present? ", os.path.isfile(current_residue_pdb_filename))
```

Comment:
looks good, but before merge, can you make this a logger comment rather than a print, plz

Reply:
oops sorry! fixing now